change/@internal-react-composites-4add5659-a0cc-45ad-9afb-18819cc01431.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Should be `prerelease`
`patch` and `minor` should only be used for stable changes.